stellar-multitenant/internal/httphandler/tenants_handler.go

```diff
@@ -29,7 +32,7 @@ type TenantsHandler struct {
 func (t TenantsHandler) GetAll(w http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
 
-	tnts, err := t.Manager.GetAllTenants(ctx)
+	tnts, err := t.Manager.GetAllTenants(ctx, nil)
```

Comment:
I think the `GET {adminServer}/tenants` should return deactivated tenants too, so the Host is able to see the IDs of tenants they want to deactivate. WDYT @ziyliu?

Would this change be enough for that?
```suggestion
	tnts, err := t.Manager.GetAllTenants(ctx, &tenant.QueryParams{})
```

Reply:
Makes sense to me, and @marwen-abid also agrees.